SQL/0000-00-00-schema.sql

```diff
@@ -1140,7 +1140,8 @@ CREATE TABLE `conflicts_resolved` (
   `OldValue2` text DEFAULT NULL,
   `NewValue` text DEFAULT NULL,
   `ConflictID` int(10) DEFAULT NULL,
-  PRIMARY KEY (`ResolvedID`)
+  PRIMARY KEY (`ResolvedID`),
+  UNIQUE KEY `ConflictID` (`ConflictID`)
```

Comment:
I'm a little confused by this.. the entry in the change log says you have 5 seconds to change it.. wouldn't that result in multiple entries for the same ConflictID in the resolved table?